src/main/resources/db/changelog/v.0.2/db.changelog-user.yaml

```diff
@@ -36,19 +36,23 @@ databaseChangeLog:
                   type: VARCHAR(50)
                   constraints:
                     nullable: false
-              - column:
-                  name: u_role_id
-                  type: UUID
-                  constraints:
-                    nullable: false
 
   - changeSet:
-      id: 04042021-add-foreignKeyConstraint-user-to-role
-      author:  maksim_maevsky
+      id: 09042021-add-type-user-role
+      author: olga_evdokimova
+      dbms: postgresql
+      sql: CREATE TYPE USER_ROLE as ENUM ('SUPER_ADMIN', 'ADMIN', 'TECH_EXPERT');
+
+  - changeSet:
+      id: 09042021-add-column-user-role
+      author: olga_evdokimova
       changes:
-        - addForeignKeyConstraint:
-            baseColumnNames: u_role_id
-            baseTableName: user
-            constraintName: fk_user_role
-            referencedColumnNames: r_id
-            referencedTableName: role
+        - addColumn:
+            tableName: user
+            columns:
```

Comment:
move this column into "createTable" section

Reply:
done